server/src/main/java/org/opensearch/action/search/SearchPhase.java

```diff
@@ -69,11 +70,26 @@ public String getName() {
     }
 
     /**
-     * Returns the SearchPhase name as {@link SearchPhaseName}. Exception will come if SearchPhase name is not defined
+     * Returns the SearchPhase name as {@link SearchPhaseName}. Exception will come if SearchPhase name is not defined.
+     * Deprecated; use getSearchPhaseNameOptional() to avoid possible exceptions.
```

Comment:
Please use the javadoc tag:

```suggestion
     * @deprecated Use getSearchPhaseNameOptional() to avoid possible exceptions.
```